### PR TITLE
Delete job applications when we delete job

### PR DESCRIPTION
**Pull request by Mari-Hayashi**

### CODE REVIEW

File: src/server/src/modules/job.ts [hunk 1]
```diff
@@ -240,11 +240,18 @@ export const updateJob = async (job: Job) => {
...
 
 /**
- * @description Deletes an existing job from the database
+ * @description Deletes an existing job and relevant job applications from the database
  * @param {number} id - Id of job to delete
  * @returns Promise
  */
-export const deleteJob = (id: Job['id']) => {
+export const deleteJob = async (id: Job['id']) => {
+    await getConnection()
```
Comment (johan1505) [inline src/server/src/modules/job.ts]: is it possible to do `await JobApplication.delete({ jobId: id });` instead?

Comment (Mari-Hayashi) [reply src/server/src/modules/job.ts]: Done.

**Review by johan1505 — APPROVED** [review]

**Comment by johan1505** [comment]
> _No description provided._

T==

File: src/server/src/modules/job.ts (hunk 1 above)
Comment (johan1505) [reply src/server/src/modules/job.ts]: There are some conflicts

**Pull request merged**